interfaces/builtin/mpris.go

```diff
@@ -221,14 +224,40 @@ func (iface *MprisInterface) ConnectedSlotSnippet(plug *interfaces.Plug, slot *i
 	}
 }
 
+func (iface *MprisInterface) GetName(attribs map[string]interface{}) (string, error) {
+	// default to snap name if 'name' attribute not set
+	mprisName := "@{SNAP_NAME}"
+	for attr := range attribs {
```

Comment:
I was on the fence with this and definitely see your POV. As a counter POV, since we are talking about security policy generation, I like to err on the side of caution and error out if we get something we don't expect. I think this also goes along with the idea that interfaces form contracts between snaps and silently ignoring part of the contract felt weird.

Thoughts?